client/spring-zeebe-starter/src/main/java/io/zeebe/spring/client/properties/ZeebeClientConfigurationProperties.java

```diff
@@ -38,6 +38,7 @@
   @Data
   public static class Message {
     private Duration timeToLive = DEFAULT.getDefaultMessageTimeToLive();
+    private Duration requestTimeout = DEFAULT.getDefaultRequestTimeout();
```

Comment:
This request timeout is not related to messages, but is the general request timeout to the zeebe cluster, therefore I would move it to the Broker class above.

Reply:
Thanks, I was really in doubt about this property. I did a quick search in the client code but it was really too fast...my fault :) I will patch in a few minutes